src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -695,6 +721,8 @@ protected void setPowThreads(int powThreads) {
         int API_PORT = 14265;
         String API_HOST = "localhost";
         List<String> REMOTE_LIMIT_API = IotaUtils.createImmutableList("addNeighbors", "getNeighbors", "removeNeighbors", "attachToTangle", "interruptAttachingToTangle");
+        InetAddress REMOTE_LIMIT_API_DEFAULT_HOST = InetAddress.getLoopbackAddress();
+        List<InetAddress> REMOTE_LIMIT_API_HOSTS = Arrays.asList(REMOTE_LIMIT_API_DEFAULT_HOST);
```

Comment:
Use `IotaUtils.createImmutableList()` please

This is a change request

Reply:
Good idea, done.